### DISCUSSION TITLE
Parking, access for customers

### DISCUSSION THREAD

**Discussion by hennzen** [streetcomplete/StreetComplete#5563] [Ideas]
When solving the Parking-fees quest for a parking lot that is for customers only (of a bar, for instance), possible answers
- yes or no
- depending on time
- max. duration

none of the answers really satisfy a possible "free, no max time, but for customers only" answer.

I would not answer this at all, because none of the answers would be accurate.

**Comment by matkoniecz** [comment]
In such cases you can leave note (or edit directly in Vespucci.

Though maybe extra answer is worth adding if that is really common.

> free, no max time, but for customers only

Is it paid for non-customers or just unavailable?

**Reply by hennzen** [reply]
It's unavailable for non-customers, in this case, and it's quite common to have customers-only parkings.

I am totally new to Street Complete and when posting this I didn't know there exist other survey questions dealing with the access tag.

So, in conjunction with the latter question, my answer to the fee-question would simply be "yes" - if I had been aware of that access tag. An extra question about access might help.

**Reply by matkoniecz** [reply]
> An extra question about access might help.

We have one!

Can you check or maybe even link this parking? Is it tagged as `access=customers` already?

( code is at https://github.com/streetcomplete/StreetComplete/blob/master/app/src/main/java/de/westnordost/streetcomplete/quests/parking_fee/AddParkingFee.kt )

**Reply by hennzen** [reply]
Ok, good point.
I checked via "iD" (browser editor) and now see that "Allowed Access" is already set to "Customers only".
Well, to recap, this seems like a beginner's issue only, in case you are not aware of at least two tags.

**Reply by matkoniecz** [reply]
> is already set to "Customers only".

Is StreetComplete showing this info when asking about fee? If no, maybe it should...

**Reply by hennzen** [reply]
> Is StreetComplete showing this info when asking about fee? If no, maybe it should...

In case of :access set to "Customers only" and :fee unset, StreetComplete does not say anything about its access when asking the fee-question. Maybe just a hint, sth. like "(independent from access") would minimize confusion for newbies.

I just found a parking lot where neither :access nor :fee is set, and StreetComplete asks about access first, which makes a lot of sense to me.
